ocaml/idl/datamodel_errors.ml

```diff
@@ -1133,8 +1133,6 @@ let _ =
     ~doc:"USB_groups are currently restricted to contain no more than one VUSB." ();
   error Api_errors.usb_already_attached [ "PUSB"; "VM" ]
     ~doc:"The USB device is currently attached to a VM." ();
-  error Api_errors.passthrough_not_enabled [ "PUSB"]
```

Comment:
I don't see this one being raised anywhere. If it's needed we need to correct the text at least.